modules/games/definitions/outwit/meta.ts

```diff
@@ -0,0 +1,17 @@
+import { OutwitDefinition } from "./_types";
+
+const outwitMeta: OutwitDefinition["meta"] = {
+  id: "outwit",
+  author: "unknown",
+  name: "Outwit",
+  tags: ["slide"],
```

Comment:
There is also the `infiltration` tag for games where the goal is to get units into a certain place!

```suggestion
  tags: ["slide", "infiltration"],
```